.env.local

```diff
@@ -4,3 +4,4 @@ AUTH_DB_VOLUME=~/haitaton-auth-data
 NGINX_CONF=nginx.conf
 FRONT_DOCKERFILE=Dockerfile-local
 ALLU_UPDATE_INITIAL_DELAY=60000
```

Comment:
Would it be better to use HAITATON_GDPR_ENABLED = false? DISABLE = false is a double negative, and sometimes they might be confusing.